cmd/notary-server/config.json

```diff
@@ -13,6 +13,6 @@
 	},
 	"storage": {
 		"backend": "mysql",
-		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
+		"db_url": "gordontheturtle:dockerrulez@tcp(notarymysql:3306)/development"
```

Comment:
I agree, this should reference the env variables, and they should be setup before this container runs.